src/ucp/wireup/wireup_cm.c

```diff
@@ -1307,9 +1371,15 @@ ucp_ep_server_init_priv_data(ucp_ep_h ep, const char *dev_name,
     ucp_context_dev_tl_bitmap(worker->context, dev_name, &ctx_tl_bitmap);
     ucp_tl_bitmap_validate(&tl_bitmap, &ctx_tl_bitmap);
 
-    status = ucp_cm_ep_priv_data_pack(ep, &tl_bitmap, UCS_LOG_LEVEL_ERROR,
-                                      sa_data_version, (void**)data_buf_p,
-                                      data_buf_size_p, ep_init_flags);
+    status = ucp_cm_ep_priv_space_is_enough(worker, ucp_ep_config(ep),
+            ep->ext->cm_idx, &tl_bitmap, UCS_LOG_LEVEL_ERROR, ep_pack_flags,
+            sa_data_version);
+    if (status != UCS_OK) {
+        goto out;
+    }
```

Comment:
why needed, won't ucp_cm_ep_priv_data_pack fail anyway?

Reply:
It could be removed. In this way, I will not use ```ucp_wireup_cm_ep_priv_space_is_enough``` in server side.